ddsketch/mapping/index_mapping.go

```diff
@@ -16,6 +16,7 @@ const (
 
 type IndexMapping interface {
 	Equals(other IndexMapping) bool
+	Copy() IndexMapping
```

Comment:
Given that mappings are immutable, why do we need `Copy()`?

Reply:
Yes, good point. Removed.